manager/integration/tests/test_node.py

```diff
@@ -350,6 +350,22 @@ def test_replica_scheduler_no_disks(client):  # NOQA
     common.wait_for_volume_delete(client, vol_name)
 
 
+@pytest.mark.node  # NOQA
+def test_disable_scheduling_on_cordoned_node(client):  # NOQA
+    """
+    Test replica scheduler: schedule replica based on
+    `Disable Scheduling On Cordoned Node` setting
+
+    1. Set `Disable Scheduling On Cordoned Node` to true.
```

Comment:
Is this a new setting which will be introduced for the users on the Longhorn UI?

Reply:
Yes, this is a new node condition we introduced. I will update the condition name from "Cordoned" to "Schedulable".